metals/src/main/scala/scala/meta/internal/metals/Indexer.scala

```diff
@@ -441,6 +450,21 @@ final case class Indexer(
                 )
               }
             }
+            // what we really want to index are "extension" methods
+            // However, we filter by `Kind.Method` because semanticdb doesn't have properties
+            // that represents "extension".
+            // Knowing `SemanticdbDefinition.foreach` uses `ScalaToplevelMtags` and it puts
+            // `Kind.Method` only to extension methods, we can safely filter extension methods
+            // by `info.kind == Kind.Method`. (TODO: add exntension properties to semanticdb schema).
```

Comment:
Is there an issue in scalameta already? If not could you raise it?

Reply:
Not yet, I'll open an issue and link from comment 👍

Reply:
https://github.com/scalameta/scalameta/issues/2799